board-flipper.user.js

```diff
@@ -35,1030 +35,106 @@
 	function boardFlipper() {
 		'use strict';
 
-		addFlipButtons();
+		document.addEventListener('keydown', (event) => event.key === 'f' && flipBoard());
+	}
+})();
+var isBoardFlipped = false;
```

Comment:
Having to keep track of `isBoardFlipped` state is a bit unfortunate. Does the `board-flipper.user.js` script not work now without it?